docs/cpp/lldb-mi.md

```diff
@@ -0,0 +1,77 @@
+---
+Order: 15
+Area: cpp
+TOCTitle: Setup Help for LLDB-MI
+ContentId: 1e84d196-397f-4dbb-9746-06f15766d83e
+PageTitle: How to setup debugging on macOS with LLDB-MI
+DateApproved: 2/01/2020
+MetaDescription: How to setup debugging on macOS with LLDB-MI
+---
+# Setup
+The debug adapter for the C/C++ extension utilizes the machine interface mode for both gdb and lldb. To use this interface in lldb, we utilize `lldb-mi`. We build and ship this executable which has a dependency on `LLDB.framework`.
+
+## Prerequisites
+The `lldb-mi` executable requires `LLDB.framework` to run.
+
+### How to obtain `LLDB.framework`
+You can get `LLDB.framework` from one of the two choices below:
+- `XCode`
+    1. Open the `Apple App Store`
+    2. Search for `XCode`
+    3. Install.
+
+- `XCode Command Line Tools`
+    1. Open a terminal
+    2. Run `xcode-select --install`
+    3. Confirm prompt
```

Comment:
`Install.` has a period but the others don't. Also is it the license prompt? maybe make this a little clearer.